gridpath/transmission/operations/carbon_emissions.py

```diff
@@ -307,6 +340,36 @@ def load_model_data(
         None: list(data_portal.data()["tx_carbon_cap_zone"].keys())
     }
 
+    # Check if timepoint emissions file exists before loading
+    timepoint_emissions_file = os.path.join(
+        scenario_directory,
+        weather_iteration,
+        hydro_iteration,
+        availability_iteration,
+        subproblem,
+        stage,
+        "inputs",
+        "transmission_timepoint_emissions.tab",
+    )
+
+    # If the timepoint emissions file exists, load the data and initialize the CRB_TX_OPR_TMPS set
```

Comment:
We don't need the "select" here. Just load the file and the parameter will be initialized for all (tx_line, tmp) combinations that we are loading data for, and take the default value for all other members of CRP_TX_OPR_TMPS.